rhaptos/cnxmlutils/tests/test_xsl_cnxml_to_html5.py

```diff
@@ -6,6 +6,7 @@
 
 from lxml import etree
 
+from .. import __version__ as version
 
```

Comment:
Should we use the actual version here, or some sort of "test" version to avoid rev'ing the expected content?

Reply:
Yep, 👍 on having a "test" version